src/devices/namron.ts

```diff
@@ -4,8 +4,8 @@ import fz from '../converters/fromZigbee';
 import tz from '../converters/toZigbee';
 import * as constants from '../lib/constants';
 import * as exposes from '../lib/exposes';
-import {forcePowerSource, light, onOff} from '../lib/modernExtend';
-import * as ota from '../lib/ota';
+import {forcePowerSource, light, onOff, ota, identify, electricityMeter} from '../lib/modernExtend';
+import * as ota_old from '../lib/ota';
```

Comment:
renamed to avoid name conflict. There is probably another best practice somewhere that I'm not aware off.